app/src/interfaces/translations/translation-form.vue

```diff
@@ -0,0 +1,186 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { isEmpty } from 'lodash';
+import { usePermissions } from '@/composables/use-permissions';
+import { type DisplayItem } from '@/composables/use-relation-multiple';
+import { type RelationM2M } from '@/composables/use-relation-m2m';
+import LanguageSelect from './language-select.vue';
+
+const {
+	languageOptions,
+	disabled,
+	relationInfo,
+	getItemWithLang,
+	displayItems,
+	fetchedItems,
+	getItemEdits,
+	isLocalItem,
+	remove,
+	updateValue,
+} = defineProps<{
+	languageOptions: Record<string, any>[];
+	disabled: boolean;
+	autofocus: boolean;
+	relationInfo?: RelationM2M;
+	getItemWithLang: (items: Record<string, any>[], lang: string | undefined) => DisplayItem | undefined;
+	loading: boolean;
```

Comment:
`loading` and `autofocus` are marked as non-optional types but are not included in the default props.

It seems you're passing them both in `app/src/interfaces/translations/translations.vue`